fractal-puppy/Engine
# Toggle billboard + custom particle orientation / Trail renderer color picker

Now the particles could be billboarded or aligned to an axis. The trails color could be modified

To test:

Particle systems:
Create particle system
In the component toggle billboarded check
Try different values to the lookAt vector. By default aligns in the world Y axis(Keep as unit vector as possible).

Trail renderers:

Create a trail render.
Pick a texture.
Pick a color.
Move it along the scene.
The weird deformations is caused by imguizmo imprecissions. Ignore it.
